apollo-ios/Sources/Apollo/CacheWriteInterceptor.swift

```diff
@@ -7,17 +7,12 @@ import ApolloAPI
 public struct CacheWriteInterceptor: ApolloInterceptor {
 
   public enum CacheWriteError: Error, LocalizedError {
-    @available(*, deprecated, message: "Will be removed in a future version.")
     case noResponseToParse
 
-    case missingCacheRecords
```

Comment:
Arguably this is a breaking change but given the short availability of release `1.14.0` I don't see this as being too impactful. Like the deprecation of `noResponseToParse` it would not have been introduced if we'd caught the bug earlier.